test/parallel/test-process-external-stdio-close-spawn.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+// Refs: https://github.com/nodejs/node/issues/947
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+
+if (process.argv[2] === 'child') {
+  process.on('message', common.mustCall((msg) => {
+    assert.strictEqual(msg, 'go');
+    console.log('logging should not cause a crash');
```

Comment:
the `console.log` message is what will cause the test to fail in case of a regression. It is the central part of the test. This is the approach in the test from which this was inspired [test/parallel/test-process-external-stdio-close.js](https://github.com/mcollina/node/blob/a04c7313f298f123fc48fc48cd57dc460addc2a7/test/parallel/test-process-external-stdio-close.js) by @addaleax.